README.md

```diff
@@ -1,2 +1,4 @@
 # Trawl
 Trawl is a WebApp to analyse your behaviour on social media sites.
+
+Hello Theresa was here.
```

Comment:
Remove that line! Add some grunt commands instead like in https://git.mediacube.at/fhs38593/ebrahimnia_mmp1/tree/master

To see the code for creating the styling you have to change to the Raw version of the Readme.md in the linked repository.